app/src/renderer/components/monitor/BlockchainSelectModal.vue

```diff
@@ -46,7 +46,7 @@ export default {
   z-index 1000
 
 .modal-blockchain-container 
-  border-top 1px solid bc
+  border-top px solid bc
```

Comment:
is it maybe worth making border a mixin?

Reply:
Would be useful to do later!